doc/source/API/LogHandler.rst

```diff
@@ -0,0 +1,17 @@
+LogHandler
+==============
+This section lists modules for creating a handler to
+send log messages in AEDT logging stream.
+
+
+.. currentmodule:: pyaedt
+
+.. autosummary::
+   :toctree: _autosummary
+   :template: custom-class-template.rst
+   :nosignatures:
+
+.. autoclass:: pyaedt.log_handler.LogHandler
+   :inherited-members: Handler
```

Comment:
@akaszynski I am not sure we can have both `autoclass `and `autosummary `in the same rst. At some point, I got some conflicts for the class and I was expected it, but then I did not get any errors all of a sudden.